specs/APIReview_AllowExternalDrop.md

```diff
@@ -0,0 +1,100 @@
+# Background
+Currently the functionality of dragging and dropping external objects(e.g. files, hyperlinks) into webview2 is default enabled in webview2 and there is no way to disable it. Some developers may want to disbale this functionality in their applications based upon their scenarios. According to such feature requirements, we add the new API to provide developers with the capability to configure the external drag&drop functionality.
+
+# Description
+We add a new `AllowExternalDrop` property in `CoreWebView2Controller`. 
+This API allows end developers to toggle the external drag&drop functionality easily.
+If it's disabled, drag&drop objects from outside into current webview2 will be forbidden.
```

Comment:
"from outside" is too vague (outside what? the current process?). "from outside the WebView" would be clearer.

Reply:
Maybe something like:
```suggestion
If it's disabled, dragging objects from outside the bounds of the CoreWebView2Controller and dropping into the CoreWebView2Controller will be disallowed.
```

Reply:
changed to `If it's disabled, dragging objects from outside the bounds of the WebView and dropping into the WebView will be disallowed`